diff -urN redhat-artwork-0.73/art/cursor/Bluecurve/Makefile.am /usr/local/src/redhat-artwork-0.73/art/cursor/Bluecurve/Makefile.am
--- redhat-artwork-0.73/art/cursor/Bluecurve/Makefile.am	Mon Feb 24 06:06:19 2003
+++ /usr/local/src/redhat-artwork-0.73/art/cursor/Bluecurve/Makefile.am	Tue Aug 12 00:40:33 2003
@@ -56,15 +56,15 @@
 clean:
 	rm -rf Bluecurve
 
-install-data-local:
-	rm -rf $(datadir)/icons/Bluecurve/cursors
-	mkdir -p $(datadir)/icons/Bluecurve/cursors
-	for i in `cd Bluecurve && echo *` ; do						\
-	  if test -L Bluecurve/$$i ; then						\
-	    cp -d Bluecurve/$$i $(datadir)/icons/Bluecurve/cursors/$$i ;		\
-	  else										\
-	    install -m 0644 Bluecurve/$$i $(datadir)/icons/Bluecurve/cursors/$$i ;	\
-          fi ;										\
-	done
+#install-data-local:
+#	rm -rf $(datadir)/icons/Bluecurve/cursors
+#	mkdir -p $(datadir)/icons/Bluecurve/cursors
+#	for i in `cd Bluecurve && echo *` ; do						\
+#	  if test -L Bluecurve/$$i ; then						\
+#	    cp -d Bluecurve/$$i $(datadir)/icons/Bluecurve/cursors/$$i ;		\
+#	  else										\
+#	    install -m 0644 Bluecurve/$$i $(datadir)/icons/Bluecurve/cursors/$$i ;	\
+#         fi ;										\
+#	done
 
 EXTRA_DIST=$(bluecurve_images) Bluecurve.cursortheme
diff -urN redhat-artwork-0.73/art/cursor/Bluecurve-inverse/Makefile.am /usr/local/src/redhat-artwork-0.73/art/cursor/Bluecurve-inverse/Makefile.am
--- redhat-artwork-0.73/art/cursor/Bluecurve-inverse/Makefile.am	Mon Feb 24 06:06:19 2003
+++ /usr/local/src/redhat-artwork-0.73/art/cursor/Bluecurve-inverse/Makefile.am	Tue Aug 12 00:40:51 2003
@@ -56,15 +56,15 @@
 clean:
 	rm -rf Bluecurve-inverse
 
-install-data-local:
-	rm -rf $(datadir)/icons/Bluecurve-inverse/cursors
-	mkdir -p $(datadir)/icons/Bluecurve-inverse/cursors
-	for i in `cd Bluecurve-inverse && echo *` ; do							\
-	  if test -L Bluecurve-inverse/$$i ; then							\
-	    cp -d Bluecurve-inverse/$$i $(datadir)/icons/Bluecurve-inverse/cursors/$$i ;		\
-	  else												\
-	    install -m 0644 Bluecurve-inverse/$$i $(datadir)/icons/Bluecurve-inverse/cursors/$$i ;	\
-          fi ;												\
-	done
+#install-data-local:
+#	rm -rf $(datadir)/icons/Bluecurve-inverse/cursors
+#	mkdir -p $(datadir)/icons/Bluecurve-inverse/cursors
+#	for i in `cd Bluecurve-inverse && echo *` ; do							\
+#	  if test -L Bluecurve-inverse/$$i ; then							\
+#	    cp -d Bluecurve-inverse/$$i $(datadir)/icons/Bluecurve-inverse/cursors/$$i ;		\
+#	  else												\
+#	    install -m 0644 Bluecurve-inverse/$$i $(datadir)/icons/Bluecurve-inverse/cursors/$$i ;	\
+#         fi ;												\
+#	done
 
 EXTRA_DIST=$(bluecurve_images) Bluecurve-inverse.cursortheme
diff -urN redhat-artwork-0.73/art/kde/kwin/Bluecurve/Makefile.am /usr/local/src/redhat-artwork-0.73/art/kde/kwin/Bluecurve/Makefile.am
--- redhat-artwork-0.73/art/kde/kwin/Bluecurve/Makefile.am	Fri Sep 13 17:07:28 2002
+++ /usr/local/src/redhat-artwork-0.73/art/kde/kwin/Bluecurve/Makefile.am	Tue Aug 12 00:49:21 2003
@@ -4,7 +4,7 @@
 
 kwin_bluecurve_la_SOURCES = redhatclient.cpp
 kwin_bluecurve_la_LIBADD = $(kde_libraries)/kwin.la
-kwin_bluecurve_la_LDFLAGS = $(all_libraries) $(KDE_PLUGIN) -module
+kwin_bluecurve_la_LDFLAGS = $(all_libraries) -module
 
 themedir = $(datadir)/apps/kwin
 theme_DATA = bluecurve.desktop
diff -urN redhat-artwork-0.73/art/pixmaps/Makefile.am /usr/local/src/redhat-artwork-0.73/art/pixmaps/Makefile.am
--- redhat-artwork-0.73/art/pixmaps/Makefile.am	Mon Jan 27 20:14:53 2003
+++ /usr/local/src/redhat-artwork-0.73/art/pixmaps/Makefile.am	Tue Aug 12 00:41:03 2003
@@ -27,9 +27,9 @@
 
 EXTRA_DIST=$(pixmap_DATA)
 
-install-data-local:
-	mkdir -p $(datadir)/icons/Bluecurve/48x48/apps; \
-	(cd $(datadir)/icons/Bluecurve/48x48/apps;	\
-	 for icon in $(pixmap_DATA); do			\
-		ln -s ../../../../pixmaps/$$icon . ;	\
-	 done);
+#install-data-local:
+#	mkdir -p $(datadir)/icons/Bluecurve/48x48/apps; \
+#	(cd $(datadir)/icons/Bluecurve/48x48/apps;	\
+#	 for icon in $(pixmap_DATA); do			\
+#		ln -s ../../../../pixmaps/$$icon . ;	\
+#	 done);
