diff -urN /home/users/builder/1orig/redhat-artwork-0.78/art/cursor/Bluecurve/Makefile.am redhat-artwork-0.78/art/cursor/Bluecurve/Makefile.am
--- /home/users/builder/1orig/redhat-artwork-0.78/art/cursor/Bluecurve/Makefile.am	Mon Feb 24 08:13:11 2003
+++ redhat-artwork-0.78/art/cursor/Bluecurve/Makefile.am	Fri Aug 15 20:13:40 2003
@@ -57,13 +57,13 @@
 	rm -rf Bluecurve
 
 install-data-local:
-	rm -rf $(datadir)/icons/Bluecurve/cursors
-	mkdir -p $(datadir)/icons/Bluecurve/cursors
+	rm -rf $(DESTDIR)$(datadir)/icons/Bluecurve/cursors
+	mkdir -p $(DESTDIR)$(datadir)/icons/Bluecurve/cursors
 	for i in `cd Bluecurve && echo *` ; do						\
 	  if test -L Bluecurve/$$i ; then						\
-	    cp -d Bluecurve/$$i $(datadir)/icons/Bluecurve/cursors/$$i ;		\
+	    cp -d Bluecurve/$$i $(DESTDIR)$(datadir)/icons/Bluecurve/cursors/$$i ;		\
 	  else										\
-	    install -m 0644 Bluecurve/$$i $(datadir)/icons/Bluecurve/cursors/$$i ;	\
+	    install -m 0644 Bluecurve/$$i $(DESTDIR)$(datadir)/icons/Bluecurve/cursors/$$i ;	\
           fi ;										\
 	done
 
diff -urN /home/users/builder/1orig/redhat-artwork-0.78/art/cursor/Bluecurve-inverse/Makefile.am redhat-artwork-0.78/art/cursor/Bluecurve-inverse/Makefile.am
--- /home/users/builder/1orig/redhat-artwork-0.78/art/cursor/Bluecurve-inverse/Makefile.am	Mon Feb 24 08:13:17 2003
+++ redhat-artwork-0.78/art/cursor/Bluecurve-inverse/Makefile.am	Fri Aug 15 20:13:40 2003
@@ -57,13 +57,13 @@
 	rm -rf Bluecurve-inverse
 
 install-data-local:
-	rm -rf $(datadir)/icons/Bluecurve-inverse/cursors
-	mkdir -p $(datadir)/icons/Bluecurve-inverse/cursors
+	rm -rf $(DESTDIR)$(datadir)/icons/Bluecurve-inverse/cursors
+	mkdir -p $(DESTDIR)$(datadir)/icons/Bluecurve-inverse/cursors
 	for i in `cd Bluecurve-inverse && echo *` ; do							\
 	  if test -L Bluecurve-inverse/$$i ; then							\
-	    cp -d Bluecurve-inverse/$$i $(datadir)/icons/Bluecurve-inverse/cursors/$$i ;		\
+	    cp -d Bluecurve-inverse/$$i $(DESTDIR)$(datadir)/icons/Bluecurve-inverse/cursors/$$i ;		\
 	  else												\
-	    install -m 0644 Bluecurve-inverse/$$i $(datadir)/icons/Bluecurve-inverse/cursors/$$i ;	\
+	    install -m 0644 Bluecurve-inverse/$$i $(DESTDIR)$(datadir)/icons/Bluecurve-inverse/cursors/$$i ;	\
           fi ;												\
 	done
 
diff -urN /home/users/builder/1orig/redhat-artwork-0.78/art/icon/Makefile.am redhat-artwork-0.78/art/icon/Makefile.am
--- /home/users/builder/1orig/redhat-artwork-0.78/art/icon/Makefile.am	Wed Aug  6 11:07:57 2003
+++ redhat-artwork-0.78/art/icon/Makefile.am	Fri Aug 15 20:54:38 2003
@@ -72,8 +72,8 @@
 												\
 		 $(INSTALL_DATA) $(srcdir)/$$THEME/index.theme $(DESTDIR)$(THEME_DIR)/$$THEME;	\
 	done ;											\
-	mkdir -p $(datadir)/pixmaps ;								\
-	(cd $(datadir)/pixmaps ;								\
+	mkdir -p $(DESTDIR)$(datadir)/pixmaps ;								\
+	(cd $(DESTDIR)$(datadir)/pixmaps ;								\
 	 for icon in $(PIXMAPS_LINKS); do							\
 		ln -s ../icons/Bluecurve/48x48/apps/$$icon . ;					\
 	 done);
@@ -110,5 +110,5 @@
 # 		(iconfiles=`find $(srcdir)/$$THEME -name "*.icon"`;			\
 # 		 for icon in $$iconfiles; do						\
 # 			cp -f $$icon $(distdir)/$$icon || exit 1;			\
-# 		 done);									\
+# 		 done);
 
diff -urN /home/users/builder/1orig/redhat-artwork-0.78/art/kde/kwin/Bluecurve/Makefile.am redhat-artwork-0.78/art/kde/kwin/Bluecurve/Makefile.am
--- /home/users/builder/1orig/redhat-artwork-0.78/art/kde/kwin/Bluecurve/Makefile.am	Fri Sep 13 17:07:28 2002
+++ redhat-artwork-0.78/art/kde/kwin/Bluecurve/Makefile.am	Fri Aug 15 20:13:40 2003
@@ -4,7 +4,7 @@
 
 kwin_bluecurve_la_SOURCES = redhatclient.cpp
 kwin_bluecurve_la_LIBADD = $(kde_libraries)/kwin.la
-kwin_bluecurve_la_LDFLAGS = $(all_libraries) $(KDE_PLUGIN) -module
+kwin_bluecurve_la_LDFLAGS = $(all_libraries) -module
 
 themedir = $(datadir)/apps/kwin
 theme_DATA = bluecurve.desktop
